src/brouwers/kitreviews/tests/test_members.py

```diff
@@ -0,0 +1,42 @@
+from django.core.exceptions import ValidationError
+from django.test import TestCase
+
+from brouwers.users.tests.factories import UserFactory
+from ..models import KitReview, KitReviewVote, MIN_RATING, MAX_RATING
+from .factories import KitReviewFactory, KitReviewVoteFactory
+
+
+class KitReviewTest(TestCase):
+    """
+    Test the correct behaviour of brouwers.kitreviews.models.KitReview.
+    """
+
+    def test_kit_review_rating(self):
+        kit_review1 = KitReviewFactory.create()
+        self.assertEqual(kit_review1.rating, 50)
```

Comment:
this test is ok-ish since it tests the default value. However, the rating could also be defined in the factory model, so in this case it might be better to test with 'KitReview.objecst.create(...)`. I don't consider these kind of tests as a requirement though.